bin/command_hvac_csc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# This file is part of ts_hvac.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the Vera Rubin Observatory
+# Project (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+
+from lsst.ts import salobj
+
+
+async def main():
+    domain = salobj.Domain()
+    havc = salobj.Remote(domain=domain, name="HVAC")
+    await havc.start_task
+    await havc.cmd_start.set_start(timeout=20)
+    await salobj.set_summary_state(remote=havc, state=salobj.State.ENABLED)
+
+
+if __name__ == "__main__":
```

Comment:
Please consider a different name, such as `enable_hvac.py`. Usually `bin/command_foo.py` runs a "Foo" CSC commander (an instance of or subclass of `salobj.CscCommander`. You might actually want that someday.

If you want a user to be able to start the HVAC in enabled state then add `enable_cmdline_state=True` as a class variable to your CSC. (I have not looked yet to see if you already did that).

Reply:
I removed the script since it was merely a mnemonic for me how to command a CSC. I know by now so I no longer need the script.